.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+Contributing
+============
+
+If you would like to contribute code you can do so through GitHub by
+forking the repository and sending a pull request.
+
+When submitting code, please make every effort to follow existing conventions
+and style in order to keep the code as readable as possible. Please also make
+sure your code compiles for your intended version of Unity.
+
+Some general advice
+
+- Don’t change public API lightly; avoid it if possible, and include your reasoning in the PR if essential.  It causes pain for developers who use Embrace and sometimes runtime errors.
+- Get working code on a personal branch with tests before you submit a PR.
+- Don't introduce new dependencies or major new functionality.
+
+## Reporting issues
+
+If you have any trouble to start contributing, please [contact us directly](mailto:support@embrace.io) or reach out in our [Community Slack](https://embraceio-community.slack.com/)
+Finally, feel free to open a [new issue](https://github.com/embrace-io/embrace-android-sdk/issues/new).
```

Comment:
We should change this to https://github.com/embrace-io/embrace-unity-sdk/issues/new